microsite/data/plugins/armo-kubescape

```diff
@@ -0,0 +1,12 @@
+title: Armo Kubescape Plugin
+author: Armo
+authorUrl: https://github.com/armosec
+category: Monitoring
+description: open-source Kubernetes Security platform, made for DevOps.
+  Our patented technology and open-source solutions fit natively within the CI/CD pipeline and existing development tools, assuring DevOps, DevSecOps, and developers that every Kubernetes cluster, container, and microservice is born and remains secure, from development to production and from configuration to run-time, every time
+documentation: https://github.com/armosec/backstage-plugin#readme
+iconUrl: https://cloud.armosec.io/assets/images/kubescape-logo.svg
+# You can provide an external url or add your logo under static/img and provide a path
+# relative to static/ e.g. img/my-logo.png
+npmPackageName: '@backstage/armo-kubescape-plugin'
```

Comment:
Looking at this closer this package - `@backstage/armo-kubescape-plugin` - doesn't exist on NPM. That makes sense as you would not be able to publish to the `@backstage` Org unless it came from this repo. You'll want to make sure to publish your package to NPM and probably under your Org - I'm thinking it would be `@armosec`.